.github/workflows/keep-alive-streamlit.yml

```
@@ -0,0 +1,21 @@
+  schedule:
+    - cron: '17 */2 * * *'
+  workflow_dispatch:
+
+jobs:
+  ping:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: pnpm install
+        uses: ./.github/actions/pnpm-install
```

actionlint (1.7.12)

[error] missing input "bryntum_access_token" which is required by action "pnpm-install" defined at "./.github/actions/pnpm-install". all required inputs are "bryntum_access_token"

(action)